djautotask/api.py

```diff
@@ -110,6 +113,8 @@ def get_zone_info(username):
         response = requests.get(endpoint_url)
         if 200 == response.status_code:
             resp_json = response.json()
+            # Whenever get_zone_info is triggered, update cache
+            update_cache(resp_json)
```

Comment:
This is not what we want to do here, get _zone_info is triggered every time we initialize a synchronizer, which is almost every time we make a request to the AT API now. There isn't a point in keeping a cache if its refreshed nearly every time.

Reply:
Oh, really? Maybe there seems to be some confusion here.

For now, the `get_zone_info` has been used as a last call to refresh cache info.

    if not api_url_from_cache or force_fetch:
        try:
            json_obj = get_zone_info(settings.AUTOTASK_CREDENTIALS['username'])
            url = json_obj[field]
        except AutotaskAPIError as e:
            raise AutotaskAPIError('Failed to get zone info: {}'.format(e))

Reply:
Maybe you are confused w/ `get_api_connection_url` and `get_zone_info`

![image](https://user-images.githubusercontent.com/73486314/140838302-d4dd9f5e-e2e7-4eaf-9772-c30f6f1964f3.png)

Please let me know if I misunderstand something. :)

Reply:
Yes, I think you are right. I believe I mixed up `get_api_connection_url` and `get_zone_info`. Sorry about that. Just the other two points about changing the cache keys and increasing the timeout to 10 minutes.

Reply:
Thanks~ It has been fixed and review again.